### ISSUE TITLE
[Auto-detection] projectorcentral.com

### ISSUE THREAD

**Issue by ShalokShalom** [darkreader/darkreader#8692]
# Broken Website Report

## Website Address

https://www.projectorcentral.com/Epson-Pro-Cinema-LS12000-Laser-Projector-Review.htm

## Steps To Reproduce

Go on the site. 

## Problematic Observation

Site is detected as dark. Site is brighter than the sun. 

## Screenshots

![Screenshot_49](https://user-images.githubusercontent.com/6344099/164088332-eb9d75fc-4b52-4793-886f-48be51119fcd.png)

## System Information

- OS:  Linux
- Browser:  FF
- Dark Reader Version:  current

**Comment by ShalokShalom** [comment]
Same here: https://heimkino-lasertv.de/2022/01/31/epson-test-ls11000-ls12000/

**Comment by q1800** [comment]
I have checked out the link in your first post and I did not encounter any such behavior. I have included a screenshot that I believe is the exact same as yours. The actual URL is: https://www.projectorcentral.com/Epson-Pro_Cinema_LS12000.htm for the exact screenshot.

![image](https://user-images.githubusercontent.com/95879668/164091649-9e391a35-bf1f-47ef-988f-a5374c9e8052.png)

I am on Windows 11 using Firefox Nightly 101. I have tested this with both Dark Reader 4.9.47 & the newly released 4.9.48.
